sdk/keyvault/keyvault-keys/test/public/utils/testAuthentication.ts

```diff
@@ -30,38 +29,23 @@ export async function authenticate(version: string, recorder: Recorder): Promise
   const keyVaultUriName = assertEnvironmentVariable("KEYVAULT_URI").match("https://(.*.net)/")![1];
   const replacedKeyVaultUriName = replaceableVariables.KEYVAULT_URI.match("https://(.*.net)/")![1];
 
-  let attestationUri: string;
```

Comment:
Are you not testing the APIs needing attestation anymore? Or did this get refactored before?